src/features/offer/components/OfferReactions/OfferReactions.native.test.tsx

```diff
@@ -16,7 +16,7 @@ const mockBookings = { ...bookingsSnap }
 const mockUseBookings = jest.fn(() => ({
   data: mockBookings,
 }))
-jest.mock('features/bookings/api/useBookings', () => ({
+jest.mock('features/bookings/api/useBookingsQuery/useBookingsQuery', () => ({
```

Comment:
```suggestion
jest.mock('features/bookings/queries/useBookingsQuery/useBookingsQuery', () => ({
```

en plus on est dans `offer` et on doit mocker un hook venant de `bookings`

donc peut etre qu'il faudrait le bouger dans les hook commun

```suggestion
jest.mock('queries/useBookingsQuery/useBookingsQuery', () => ({
```